### PR TITLE
ADD: get all file ids

### PR DESCRIPTION
**Pull request by realemx**

### CODE REVIEW

File: src/main/java/mops/persistence/FileRepository.java [hunk 1]
```diff
@@ -157,6 +159,27 @@ public boolean fileExist(long fileId) throws StorageException {
...
+            Iterable<Result<Item>> results = minioClient.listObjects(configuration.getBucketName());
+            Set<Long> ids = new java.util.HashSet<>();
+            for (Result<Item> item : results) {
+                ids.add(
+                        Long.parseLong(item.get().objectName())
+                );
+            }
+            return ids;
+        } catch (Exception e) {
+            throw new MopsException("Fehler beim laden aller File IDs.", e);
```
Comment (Segelzwerg) [inline src/main/java/mops/persistence/FileRepository.java]: Shouldn't we throw DatabaseExpcetion?

File: src/main/java/mops/persistence/FileRepository.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import io.minio.errors.*;
 import io.minio.messages.Item;
 import lombok.extern.slf4j.Slf4j;
+import mops.exception.MopsException;
```
Comment (Segelzwerg) [inline src/main/java/mops/persistence/FileRepository.java]:
```suggestion
import mops.exception.MopsException;
import mops.businesslogic.exception.DatabaseException;
```

File: src/main/java/mops/persistence/FileRepository.java [hunk 3]
```diff
@@ -157,6 +159,27 @@ public boolean fileExist(long fileId) throws StorageException {
...
+    /**
+     * Fetches all IDs.
+     *
+     * @return all File IDs
+     */
+    @SuppressWarnings({ "PMD.AvoidCatchingGenericException", "PMD.LawOfDemeter" })
+    public Set<Long> getAllIds() throws MopsException {
+        try {
+            Iterable<Result<Item>> results = minioClient.listObjects(configuration.getBucketName());
+            Set<Long> ids = new java.util.HashSet<>();
```
Comment (iTitus) [inline src/main/java/mops/persistence/FileRepository.java]: use an import here

File: src/main/java/mops/persistence/FileRepository.java (hunk 2 above)
Comment (Segelzwerg) [reply src/main/java/mops/persistence/FileRepository.java]: Maybe we don't need this. Check are not failing. I'm confused.

File: src/main/java/mops/persistence/FileRepository.java [hunk 4]
```diff
@@ -157,6 +159,27 @@ public boolean fileExist(long fileId) throws StorageException {
...
+            for (Result<Item> item : results) {
+                ids.add(
+                        Long.parseLong(item.get().objectName())
+                );
+            }
+            return ids;
+        } catch (Exception e) {
+            throw new DatabaseException("Fehler beim laden aller File IDs.", e);
```
Comment (iTitus) [inline src/main/java/mops/persistence/FileRepository.java]:
```suggestion
            throw new DatabaseException("Fehler beim Laden aller File IDs.", e);
```

**Pull request merged**